src/main/resources/edu/cornell/kfs/module/purap/document/datadictionary/RequisitionDocument.xml

```diff
@@ -28,7 +28,7 @@
            </list>
        </property>
    </bean>
-   
+
```

Comment:
If you're not making any other updates to this file, could you undo this whitespace-only change?